alacritty_terminal/src/term/mod.rs

```diff
@@ -901,6 +902,11 @@ impl<T> Term<T> {
             }
         }
         self.visual_bell.update_config(config);
+        if config.scrolling.faux_multiplier().unwrap_or_else(|| config.scrolling.multiplier()) > 0 {
```

Comment:
We don't want to fallback to `multiplier` here. We just need to check if the faux multiplier is `0`.

We also don't want to remove it otherwise, the default should be to enable it, I'm thinking more like this:

```rust
if let Some(0) == config.scrolling.faux_multiplier() {
    self.mode.remove(TermMode::ALTERNATE_SCROLL);
}
```

The `else` is irrelevant since we set `ALTERNATE_SCROLL` by default.